### PR TITLE
stub for ReactNativeModule

### PR DESCRIPTION
**Pull request by rasom**
status: ready

### CODE REVIEW

File: src/status_im/ui/screens/accounts/events.cljs [hunk 1]
```diff
@@ -97,10 +96,11 @@
...
 (register-handler-fx
   :add-account
-  (fn [{{:keys [network] :as db} :db} [_ {:keys [address] :as account}]]
+  (fn [{{:keys [network] :as db} :db} [_ {:keys [address] :as account} password]]
     (let [account' (assoc account :network network)]
-      {:db            (assoc-in db [:accounts/accounts address] account')
-       ::save-account account'})))
+      {:db             (assoc-in db [:accounts/accounts address] account')
+       ::save-account  account'
+       :dispatch-later [{:ms 400 :dispatch [:login-account address password true]}]})))
```
Comment (oskarth) [inline src/status_im/ui/screens/accounts/events.cljs:103]: Why are we doing this? What does this have to do with stubbing status-go?

**Review by oskarth — APPROVED** [review]

File: src/status_im/ui/screens/accounts/events.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/ui/screens/accounts/events.cljs:103]: Hm, good question, i doubt that this is a good solution at all, but i'll describe the problem.

This actually fixes (or minimizes probability of reproduction) the heisenbug which happens from time to time on login: 
1) [`:show-mnemonic`](https://github.com/status-im/status-react/blob/89ffb8ba6d3882ef7fbc218e8231e0dfa3ee4d8c/src/status_im/ui/screens/accounts/events.cljs#L58) here causes some inserts/updates to db because it adds new messages to Console chat
2) [`:login-account`](https://github.com/status-im/status-react/blob/89ffb8ba6d3882ef7fbc218e8231e0dfa3ee4d8c/src/status_im/ui/screens/accounts/login/events.cljs#L44) event calls `::login` effect, which under the hood calls [`status-im.data-store.realm.core/close-account-realm`](https://github.com/status-im/status-react/blob/23399c63e172479876fe2a76b4eb9f5b9785f9b5/src/status_im/data_store/realm/core.cljs#L44) which reset `account-realm` to `nil`
3) when updates to db from 1) are executed `account-realm` (which can be `nil`) is used
4)  exception is thrown 

This rarely happens when status-go is used because app is much slower and race condition doesn't happen most of time. But without status-go this bug will be reproduced in 99% cases.

**Review by flexsurfer — APPROVED** [review]
